src/data/grammar.json

```diff
@@ -62,7 +62,7 @@
         "#titleMoney#: #investType# #card# #investReason#",
         "#titleMoney#: #investType# #card#",
         "#titleMoney#: #pro# is #investType#ing #broadCardClassification.capitalizeAll#; Should You?",
-        "BREAKING: Prices of #card.capitalizeAll# Spiking; Market Cornered by #pros#",
```

Comment:
I guess we have to choose between whether we want "Prices of a HP coffee-stained Squire Spiking; Market Cornered By LSV" or "Prices of A HP Coffee-Stained Squire Spiking; Market Cornered By LSV"; neither are ideal, but do we prefer the former over the latter? Or since that's a rare case, are we happier with the common case "Wall of Frost" rather than "Wall Of Frost". I suppose I'm saying I agree, just want to make sure we're aware of the implications.